docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -291,7 +291,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``pad``                     | P                               |                                  | See ``ffill``                                      |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``pct_change``              | N                               |                                  |                                                    |
+| ``pct_change``              | P                               |``limit``, ``freq``               |                                                    |
```

Comment:
Can we also say it's "N" for axis=1.

Reply:
N/A now since I just added axis=1 support :)